EIPS/eip-3540.md

```diff
@@ -125,7 +125,7 @@ For clarity, the *container* refers to the complete account code, while *code* r
 
 1. `JUMPDEST`-analysis is only run on the *code*.
 2. Execution starts at the first byte of the *code*, and `PC` is set to 0.
-3. If `PC` goes outside the code section bounds, execution aborts with failure.
+3. Execution stops if `PC` goes outside the code section bounds.
```

Comment:
@axic: should update motivation to not mention benefit of knowing last instruction is terminating.